### DISCUSSION TITLE
Button remove gap

### DISCUSSION THREAD

**Discussion by DasLixou** [AvaloniaUI/Avalonia#8381] [Q&A] [answered]
![grafik](https://user-images.githubusercontent.com/82600264/175560500-f3c00541-23b6-4918-92a6-3e60e9fea383.png)

Here i have a button with my prefered sizings:

```Background="#EF4444" CornerRadius="0" HorizontalAlignment="Right" Click="CloseInteraction" Width="50" HorizontalContentAlignment="Center"```

but for some reason there is a little gap between the end of the control and the button, even with the horizontalAlignment. How can i remove it? thanks :D

**Comment by timunie** [comment]
There seems to be some Margin or Spacing applied. You can use dev tools via F12 in debug mode

**Reply by DasLixou** [reply]
There isn't. there is a Padding but when i set it to 0 the red only gets smaller. WHen i increase it the size doesn't get bigger.

**Reply by timunie** [reply]
That's strange. To investigate this further we need to see the xaml of the view. The sample needs to contain both controls and the surrounding panel or grid

**Comment by DasLixou** [comment, marked as answer]
Solved it: `BorderThickness="0"`

**Reply by timunie** [reply]
Ah yeah great. Thanks for sharing your solution if anyone else is wondering what is going on 👍
